test/server/server_corpus/clusterfuzz-testcase-minimized-server_fuzz_test-5632902623657984

```diff
@@ -1 +1 @@
-static_resources {   clusters {     name: " "     connect_timeout {       nanos: 6     }     hosts {       pipe {         path: " "       }     }     health_checks {       timeout {         nanos: 6       }       interval {         nanos: 6       }       unhealthy_threshold {       }       healthy_threshold {       }       tcp_health_check {       }     }   } } cluster_manager {   upstream_bind_config {     source_address {       address: "0.0.0.0"       port_value: 0     }     freebind {       value: true     }   } } admin {   address {     pipe {       path: " "     }   } } 
+static_resources {   clusters {     name: " "     connect_timeout {       nanos: 6     }     hosts {       pipe {         path: " "       }     }     health_checks {       timeout {         nanos: 6       }       interval {         nanos: 6       }       unhealthy_threshold {       }       healthy_threshold {       }       tcp_health_check {       }     }   } } cluster_manager {   upstream_bind_config {     source_address {       address: "0.0.0.0"       port_value: 0     }     freebind {       value: true     }   } } admin {   address {     pipe {       path: " "     }   } }
```

Comment:
@asraa am i correct that its okay to remove the trailing whitespace from these fuzz test data files ?

Reply:
nm - i have excluded all of the `*_corpus/*` files from linting